pkg/apis/pipeline/v1beta1/pipelinerun_validation_test.go

```diff
@@ -370,9 +370,10 @@ func TestPipelineRun_Validate(t *testing.T) {
 
 func TestPipelineRunSpec_Invalidate(t *testing.T) {
```

Comment:
similar comment as for TaskRun, could you add a test case where the ref is valid?

Reply:
Added a test for taskref with resolver field + alpha gate and a test for taskref with resolver field + resource fields + alpha gate.